cves/CVE-2013-0305.yml

```diff
@@ -74,6 +85,7 @@ vccs:
   note: This VCC was discovered automatically via archeogit.
 - commit: 75a1aaa1f9341e558b6efe9227cf663d55704469
   note: This VCC was discovered automatically via archeogit.
+
```

Comment:
Some notes on the fix commits or additional information in another section would be great to understand how they addressed this vulnerability.

Reply:
Added more info for each commit